.github/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,5 @@
+# Code of conduct
+
+Respect that people have differences of opinion.
+
+Every piece of work balances the best outcome against a variety of constraints.
```

Comment:
Shall we get this PR in this sprint as it creates a baseline across various files? We can create a follow-up issues to expand on bits e.g. one for the CoC and another for looking into GitHub's new multiple issue templates.

Reply:
I agree with Toby, this feels to small to be useful. I would be happy to start with the [Contributor Covenant Code of Conduct](https://www.contributor-covenant.org/version/1/4/code-of-conduct.md) as it stands, then we can create a follow-up issue to consider if we wanted to deviate from this in some way.

Reply:
I think what’s missing here from Github’s criteria is ‘outlines procedures for addressing problems’. 

Github has a grievance process which we could direct contributors to, should we try mocking up what this could look like?

Reply:
@sl33p To clarify, what's missing from our existing CoC in this PR from Github’s criteria is ‘outlines procedures for addressing problems’.

I would like to address this by directing to Github's grievance process and feel that satisfies the acceptance criteria of this issue.

Reply:
I've updated the PR to incorporate the intended changes communicated above.

As GitHub now includes an Acceptable Use policy and grievance process, the intention is to move on from including those details in the CoC, but instead directs contributors to a consistent set of GitHub guidelines and processes.

Reply:
When using GitHub you are already agreeing to the Acceptable Use policy, I'm not sure if linking to this adds any real value ヽ(。_°)ノ

A code of conduct goes beyond this to outline clear guidelines on how we can help to create a positive environment, specific examples of unacceptable behaviour, our responsibilities, and the scope of the CoC. 

As I said last week, I think we should start with adopting the Contributor Covenant Code of Conduct as it stands, then we can create a follow-up issue to consider if we wanted to deviate from this in some way.